README.md

```diff
@@ -79,6 +86,17 @@ npx repo2pdf
 ### Installing and Using repo2pdf by Cloning the Repository
 
 This method involves manually cloning the repo2pdf repository and setting it up on your local machine.
+0. Dependencies:
```

Comment:
0 isnt possible. Also this isnt friendly for each system so will remove and clean up docs later today